### ISSUE TITLE
Models results not accurate

### ISSUE THREAD

**Issue by karimkalimu** [MadryLab/robustness#14]
The CIFAR10 (ResNet50 )pre-trained model (ε = 0.0) has to be 95.25% but I got 68.51% .

This code to test the model, I used mean and std values from (https://github.com/MadryLab/robustness/blob/master/robustness/datasets.py) :

!wget https://www.dropbox.com/s/yhpp4yws7sgi6lj/cifar_nat.pt?dl=0
!mv cifar_nat.pt\?dl=0 natural_model.pt

```
import torch
from robustness.datasets import CIFAR
from robustness.model_utils import make_and_restore_model
ds = CIFAR('./data/cifar')
model, _ = make_and_restore_model(arch='resnet50', dataset=ds,
             resume_path='natural_model.pt')

test_loader_cifar = torch.utils.data.DataLoader(datasets.CIFAR10(
    root='./data/cifar',train=False, download=True,transform=transforms.Compose(
        [transforms.ToTensor(),transforms.Normalize([0.4914, 0.4822, 0.4465], [0.2023, 0.1994, 0.2010])])), batch_size = 60)


test_acc = 0
model.eval()

for batch_idx, (data, target) in enumerate(test_loader_cifar):
  
  target = target.cuda()
  data = data.cuda()

  output, _ = model(data)
  result = np.where(output.max(1)[1].cpu() == target.cpu())
  test_acc += result[0].shape[0]

  
print(" test_acc = ", test_acc/10000)

```
Output : 
 test_acc =  0.6851

**Comment by dtsip** [comment]
You do not need to normalize during testing, the model already takes care of that.

Can you try again without input normalization?

**Comment by karimkalimu** [comment]
Yes it worked, I didn't know the model  take care of the input
 Thanks
